.github/workflows/changelog_verifier.yml

```diff
@@ -13,7 +13,19 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           ref: ${{ github.event.pull_request.head.sha }}
-
       - uses: dangoslen/changelog-enforcer@v3
+        id: verify-changelog-3x
+        with:
+          skipLabels: "autocut, skip-changelog"
+          changeLogPath: 'CHANGELOG-3.0.md'
```

Comment:
@kotwanikunal curious if we should be using `CHANGELOG-3.x.md` since it will be moving target essentially (like `CHANGELOG-2.x.md` which would correspond to 2.14.0 as of today)

Reply:
Sounds like a discussion for the other PR. Let me move this commit onto that branch and we can finalize the file names/convention on that thread.